wiki/Twitter/ja.md

```diff
@@ -0,0 +1,13 @@
+# Twitter
+
+## 私達はツイートもします！
+
+osu!にはいくつかのTwitterアカウントがあり、それぞれ特定の目的で運用されています。ほとんどはトラフィックが非常に少ないですが、osu!エコシステムの外部に情報を提供すのに役立ちます。
+
+| アバター | ハンドル | 説明 |
+| :-: | :-: | --- |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | ニュースとお知らせの公式ソース。トラフィックの少ない高品質のコンテンツ。 |
```

Comment:
"avatar" should be translated, at the beginning of each of these lines

Reply:
Translated it.
Thank you for the review.